CHANGELOG.md

```diff
@@ -1,4 +1,16 @@
-## 2.20.2 (Unreleased)
+## 2.20.3 (Unreleased)
+### Notes:
+- Supported Controller version: **UserConnect-6.5.2721**
+- Supported Terraform version: **v1.x**
+
+#### Multi-Cloud Transit
+1. Implemented support for Transit FireNet for AWSChina
+
+### Bug Fixes:
+1. Fixed issue where Terraform Plan shows diff for a use case of **aviatrix_transit_external_device_conn** upgraded from 6.3.c- version
```

Comment:
I believe the version was 6.4

Reply:
I think its better summarize this as a fix for when users upgrade Controller from 6.4.x to 6.5, rather than give explicit version numbers, since there are no letter versions (internal)